src/screens/Followings/elements/RenderList/index.js

```diff
@@ -0,0 +1,157 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Image, StyleSheet, Text, View} from 'react-native';
+import GlobalButton from '../../../../components/Button/GlobalButton';
+
+import {colors} from '../../../../utils/colors';
+import {fonts} from '../../../../utils/fonts';
+
+// data needed name, description, image
+const styles = StyleSheet.create({
+  buttonFollow: {
+    width: 88,
+    height: 36,
+    flexDirection: 'row',
+    backgroundColor: colors.bondi_blue,
+    borderRadius: 8,
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  textButtonFollowing: {
+    fontSize: 12,
+    color: colors.bondi_blue,
+    fontFamily: fonts.inter[600],
+    fontWeight: 'bold'
+  },
+  textButtonFollow: {
+    fontSize: 12,
+    color: colors.white,
+    fontFamily: fonts.inter[600],
+    fontWeight: 'bold'
+  },
+  profilepicture: {
+    width: 48,
+    height: 48,
+    borderColor: colors.lightgrey,
+    borderRadius: 24,
+    borderWidth: 1,
+    // backgroundColor: colors.bondi_blue,
+    resizeMode: 'cover'
+  },
+  wrapProfile: {
+    flex: 1,
+    width: '100%',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingVertical: 5
+  },
+  imageProfile: {
+    width: 48,
+    height: 48,
+    borderRadius: 48
+  },
+  wrapTextProfile: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    marginLeft: 12
+  },
+  textProfileUsername: {
+    fontFamily: fonts.inter[500],
+    fontWeight: 'bold',
+    fontSize: 14,
+    color: colors.black,
+    lineHeight: 16.94
+  },
+  textProfileFullName: {
+    fontSize: 12,
+    color: colors.gray,
+    fontFamily: fonts.inter[400],
+    lineHeight: 18,
+    flexWrap: 'wrap'
+  },
+  buttonFollowing: {
+    width: 88,
+    height: 36,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderColor: colors.bondi_blue,
+    borderWidth: 1,
+    borderRadius: 8
+  },
+  card: {
+    height: 68,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    width: '100%',
+    marginVertical: 10
+  }
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles for the `DomainList` component seem to be well defined. However, consider moving these styles into a separate file to improve modularity and maintainability. This will make the component file cleaner and easier to read, and the styles can be reused in other components if necessary.
```diff
- import {Image, StyleSheet, Text, View} from 'react-native';
+ import {Image, Text, View} from 'react-native';
  import GlobalButton from '../../../../components/Button/GlobalButton';
  
  import {colors} from '../../../../utils/colors';
  import {fonts} from '../../../../utils/fonts';
+ import styles from './DomainList.styles';
  
- // data needed name, description, image
- const styles = StyleSheet.create({
-  buttonFollow: {
-    width: 88,
-    height: 36,
-    flexDirection: 'row',
-    backgroundColor: colors.bondi_blue,
-    borderRadius: 8,
-    alignItems: 'center',
-    justifyContent: 'center'
-  },
-  ...
- });
```
Create a new file `DomainList.styles.js` in the same directory with the following content:
```javascript
import {StyleSheet} from 'react-native';
import {colors} from '../../../../utils/colors';
import {fonts} from '../../../../utils/fonts';

const styles = StyleSheet.create({
  buttonFollow: {
    width: 88,
    height: 36,
    flexDirection: 'row',
    backgroundColor: colors.bondi_blue,
    borderRadius: 8,
    alignItems: 'center',
    justifyContent: 'center'
  },
  ...
});

export default styles;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->